docs/src/content/docs/ops/backup-and-restore/backup-practices.adoc

```diff
@@ -0,0 +1,177 @@
+You can find provider-specific documentation here:
+
+* link:https://docs.aws.amazon.com/AmazonS3/latest/userguide/DataDurability.html[Amazon S3 Durability^]
+* link:https://learn.microsoft.com/en-us/azure/storage/common/storage-redundancy#durability-and-availability-parameters[Azure Storage Durability^]
+* link:https://cloud.google.com/storage/docs/availability-durability[GCS Durability^]
+
+However, these guarantees do not protect against operational risks, such as:
+
+* Accidental or malicious deletion
+* Misconfigured lifecycle policies or IAM roles
+* Loss of access due to control-plane outages or permission changes
+
+To mitigate these risks, enable the following object store features where available:
+
+* **Versioning** — retains previous versions of objects, allowing recovery in case of deletion or unintended modification.
+  - While XTDB uses immutable files, versioning remains an important safeguard and is a requirement of most available cloud backup tools.
+  - **S3**: link:https://docs.aws.amazon.com/AmazonS3/latest/userguide/Versioning.html[S3 Versioning^]
+  - **Azure Blobs**: link:https://learn.microsoft.com/en-us/azure/storage/blobs/versioning-overview[Blob Versioning^]
+  - **GCS**: link:https://cloud.google.com/storage/docs/object-versioning[Object Versioning^]
+* **Soft delete** — temporary protection against deletion
+  - Typically paired with versioning or retention policies.
+  - **S3**: _(Handled implicitly by versioning — deleting an object adds a delete marker, allowing recovery of previous versions)_
+  - **Azure Blobs**: link:https://learn.microsoft.com/en-us/azure/storage/blobs/soft-delete-container-overview[Soft delete for containers^] 
+  - **GCS**: link:https://cloud.google.com/storage/docs/soft-delete[Soft delete^]
+* **Replication** — cross-region replication for resilience against regional failure
+  - **S3**: link:https://docs.aws.amazon.com/AmazonS3/latest/userguide/replication.html[S3 Replication^]
+  - **Azure Blobs**: link:https://learn.microsoft.com/en-us/azure/storage/common/storage-redundancy[Zone Redundant / Geo-Redundant Storage^]
+  - **GCS**: link:https://cloud.google.com/storage/docs/locations#considerations[Dual-region / multi-region buckets^]
+
+These features act as a first line of defence — allowing recovery even before formal backups are consulted.
+
+=== Full Backups for Point-in-Time Recovery
+
+Full backups of the storage module are the recommended approach for recovering to a known good point in time, migrating XTDB environments, or in the case of a complete loss of object store.
+
+Because storage files are immutable once written, they are ideally suited to snapshot-based backup strategies.
+
+To perform a full backup:
+
+* Capture the full object store prefix or container used by XTDB, including all files written to date
+* Ensure the backup includes all files associated with the latest indexed block
```

Comment:
Having this as a separate bullet point makes it sound unrelated to the first, but IIUC it's definitely the same process, right? In which case perhaps you can just have a single point which includes a rephrased form of this sentence which provides some understanding/reassuring, e.g. "By including all files under the prefix (including some non-essentially files), this will reliably ensure that the backup includes all files associated with the latest indexed block"